src/worker.py

```
@@ -1329,6 +1331,321 @@ async def serve_static(path: str, env):
+                    sid = data.get("session_id", str(uuid.uuid4()))
+                    self.sessions[sid] = {
+                        "ws": ws,
+                        "participant_id": data.get("participant_id", "unknown"),
+                        "display_name": data.get("display_name", "Unknown"),
+                        "position": data.get("position", {"x": 400, "y": 300}),
+                        "direction": data.get("direction", "down"),
+                        "is_moving": False,
+                        "seat_id": data.get("seat_id", ""),
+                    }
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)